### PR TITLE
changed sql query for finReport top products

### PR DESCRIPTION
**Pull request by JGray5280**

### CODE REVIEW

**Comment by JGray5280** [comment]
removed credentials and fixed bug for top product on fin report. The fin report and marketing are both now fully functional.

File: FinReport.py [hunk 1]
```diff
@@ -119,13 +119,16 @@ def finReport():
...
-          WHERE [date] >= '{deltaTime}' AND
-                [date]   <= '{parseTime}' 
-         GROUP BY product_id,sale_price
-         ORDER BY product_count DESC
+        cursor.execute(f"""
+        SELECT TOP({numResults}) customer_orders_team2.product_id,sale_price,
+        SUM([product_quantity]) AS product_count
+        FROM (inventory_team2
+        INNER JOIN customer_orders_team2 ON customer_orders_team2.product_id = inventory_team2.product_id
+        )
```
Comment (Diemolin) [inline FinReport.py:127]: Is this where it grabs the top orders?

**Review by Diemolin — APPROVED** [review]
Looks good and it print out the values as well

**Review by Gaunkle — APPROVED** [review]
Looks like a clean-up of the small bug. Also removing the hard coded credentials

**Pull request merged**